docs/openapi/components/schemas/common/MinesActPermit.yml

```
@@ -0,0 +1,177 @@
+---
+$linkedData:
+  term: MinesActPermit
+  '@id': https://w3id.org/traceability#MinesActPermit
+title: Mines Act Permit
+description: The Mines Act regulates mining activities in British Columbia (B.C.) - from early exploration to development, production, reclamation, closure, and post-closure. A permit must be in place before any work in, on, or about a mine can occur. A B.C. Mines Act Permit credential issued according to this Governance Framework (GF) provides evidence the credential holder has been issued a B.C. Mines Act Permit.
+type: object
+properties:
+  type:
+    type: array
+    readOnly: true
+    const:
+      - MinesActPermit
+    default:
+      - MinesActPermit
+    items:
+      type: string
+      enum:
+        - MinesActPermit
+  permit_no:
+    title: Permit Number
+    description: >-
+      (3-digit) Alpha-numeric identifier: M-##- or C-##- Permit Numbers are assigned to a specific mining operation or activity that has been granted permission or authorization. Mines Act Section 10 (1).
```

3-digit, so shouldn't there be 3 numeric placeholders?
```suggestion
      (3-digit) Alpha-numeric identifier: M-###- or C-###- Permit Numbers are assigned to a specific mining operation or activity that has been granted permission or authorization. Mines Act Section 10 (1).
```

---

I will request an example to find out and confirm the exact form of the value.